api/v1beta1/helmchart_types.go

```diff
@@ -25,6 +25,20 @@ import (
 // HelmChartKind is the string representation of a HelmChart.
 const HelmChartKind = "HelmChart"
 
+// ReconcileStrategy determines what enables reconciliation.
+// ChartVersion reconciles when the version of the Helm Chart is different.
+// Revision reconciles when the Revision of the source is different.
+// +kubebuilder:validation:Enum=ChartVersion;Revision
+type ReconcileStrategy string
```

Comment:
Not sure if this is a good idea to use an alias for the string. We have to import the source API into the helm-controller API to compare types.